Kconfig

```diff
@@ -259,4 +259,10 @@ config DEBUG_IPC_COUNTERS
 	help
 	  Select for enabling tracing IPC counter in SRAM_REG mailbox
 
+config PERF_CNT
```

Comment:
In guideline `spartan` applies only to local code, kconfig symbols are global and shouldn't have shortcuts like `CNT`, acronyms are OK tho.

Reply:
Name changed to PERFORMANCE_COUNTERS

Reply:
@mmaka1 thanks